deployment/docker-compose.yml

```diff
@@ -35,7 +35,7 @@ db:
   container_name: mezzanine-db
   image: kartoza/postgis:9.4-2.1
   volumes:
-    - ./pg/postgres_data:/var/lib/postgresql
+#    - ./pg/postgres_data:/var/lib/postgresql
```

Comment:
If we are going to do this, we have to know that our backups are good and reliable. Until then I think it is better to persist to disk if possible.